modules/GenericDataParser/src/server/generic_data_parser.R

```diff
@@ -3862,7 +3928,8 @@ parseGenericData <- function(request) {
   # This is used for development: outputs the JSON rather than sending it to the
   # server and does not wrap everything in tryCatch so debug will keep printing
   developmentMode <- messenger()$devMode
-  
+  # developmentMode<-TRUE
```

Comment:
nit: Can we clean up this commented out line?

Reply:
Removed